docs/compliance/0-Setup-AdministrativeAccountAndPermission.md

```diff
@@ -0,0 +1,82 @@
+# Automated Foundational Architecture for PCI DSS-Compliant Environments (v1)
+
+## Script Details: `0-Setup-AdministrativeAccountAndPermission.ps1`
+
+This PowerShell script is used to verify pre-deployment requirements for the Payment Card Payment processing solution for PCI DSS enablement.
+ 
+# Description 
+ This Powershell script automates the installation and verification of the PowerShell modules, as well as configuring the administrative user of the solution. 
+ > NOTE: This script MUST be run as *Local Administrator* with elevated prividges. For more information ,see [Why do I need to run as local administrator?](https://social.technet.microsoft.com/Forums/scriptcenter/en-US/41a4ba3d-93fd-485b-be22-c877afff1bd8/how-to-run-a-powershell-script-in-admin-account?forum=ITCG)  
+
+ Running this script is not required, but installation will fail if the following modules have not been properly configured:
+- AzureRM
+- AzureAD
+- MSOnline
+- AzureDiagnosticsAndLogAnalytics
+- SqlServer
+- Enable-AzureRMDiagnostics (Script)
+
+## Example 1: Installing required modules
+
+```powershell
+.\0-Setup-AdministrativeAccountAndPermission.ps1 -installModules
+```
+This command will validate or install any missing PowerShell modules which are required for this foundational architecture.
+
+## Example 2: Configuring your global admin
+
+```powershell
+.\0-Setup-AdministrativeAccountAndPermission.ps1 
+    -azureADDomainName contosowebstore.onmicrosoft.com
+    -tenantId XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX
+    -subscriptionId XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX
+    -configureGlobalAdmin 
+ ```
+
+ This command will deploy installed modules, and setup the solution on a **new subscription**. It will create the user `adminXX@contosowebstore.onmicrosoft.com` with a randomly generated strong password (15 characters minimum, with uppercase and lowercase letters, and at least one number and and one special character.) 
```

Comment:
- What is the requirement for a new subscription?
- Does the domain name need to be associated with your AAD tenant?